content/docs/csidriver/features/powerscale.md

```diff
@@ -527,3 +527,10 @@ In the case of operator-based installation, default permission for powerscale di
 
 Other ways of configuring powerscale volume permissions remain the same as helm-based installation.
 
+## PV/PVC Metrics
+
+CSI Driver for Dell EMC Powerscale 2.1.0 and above supports volume health monitoring.This allows Kubernetes to report on the condition of the underlying volumes via events when a volume condition is abnormal. 
```

Comment:
use PowerScale instead of Powerscale , S to be caps

Reply:
Addressed